cli.js

```diff
@@ -2,11 +2,19 @@
 'use strict';
 const meow = require('meow');
 const getPort = require('get-port');
+const meowTemplate = require('./meow-template');
```

Comment:
It should be in the same file.

Reply:
I moved it to make an exclusive config for this file in `.editorconfig`, four space tabs on CLI is taking a lot of screen space